package.json

```diff
@@ -17,8 +17,8 @@
   "author": "Paul Grau <paul@blocko.io>",
   "license": "ISC",
   "dependencies": {
-    "@herajs/client": "^1.2.1",
-    "@herajs/common": "^1.2.1",
+    "@herajs/client": "^2.0.1",
+    "@herajs/common": "^2.0.1",
```

Comment:
review the 2 pull requests at the `herajs` repo first
a new version of `herajs` must be build with those PRs, and released
then the new version can be used here